README.md

```diff
@@ -95,3 +95,12 @@ npm run dev
 
 To build the distributable (application bundle on macOS, installer on Windows),
 run `npm run build`.
+
+### How To Test
+
+Use the following commands to run unit tests and e2e tests.
+
+```
+npm test
+npm run e2e
```

Comment:
`npm run e2e` doesn't work:
```
$ npm run e2e
npm ERR! Missing script: "e2e"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run
```

Should be:
`npm run test:e2e`

Reply:
Updated the instructions in README.md in 97469e8